src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -936,6 +936,15 @@ private DetectProperties() {
             .setGroups(DetectGroup.IMPACT_ANALYSIS, DetectGroup.GLOBAL, DetectGroup.SOURCE_SCAN)
             .build();
 
+    public static final BooleanProperty DETECT_COMPONENT_LOCATION_ANALYSIS_ENABLED =
+            BooleanProperty.newBuilder("detect.component.location.analysis.enabled", false)
```

Comment:
Leaving this as something to revisit in the feature branch before merging overall feature to master so that this isn't blocking part 2. Will discuss with team and PM on the customer facing wording we wish to use (for example something like **detect.component.locator**)